example/lib/navigation/navigator_bloc.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:async';
+import 'package:blemulator_example/peripheral_details/peripheral_details_screen.dart';
+import 'package:bloc/bloc.dart';
+import 'package:flutter/material.dart';
+import './bloc.dart';
+
+class NavigatorBloc extends Bloc<NavigatorEvent, dynamic> {
```

Comment:
it is not tested

Reply:
It was discussed to create a separate issue for this task.